### DISCUSSION TITLE
Vercel visitation

### DISCUSSION THREAD

**Discussion by emon3455** [vercel/community#3851] [Help]
### Summary

I cannot reach the vercel.com website with my network but is I use VPN it is accessible. and because of this my deploy server is not working. I need help.

### Example

https://vercel.com/dashboard

### Steps to Reproduce

i cannot reach the vercel.com website. it is loading and loading and my all previous deploy server is not working. but when i use vpn i can access the vercel.com without vpn i am not able to access that please help me out

**Comment by amyegan** [comment]
Hey @emon3455. Can you tell us more about the network where the site won't load?

**Reply by emon3455** [reply]
I cannot visit vercel website as well as the dashboard also my deployed project is also not working. And then i tryed VPN and then i can access the website of vercel and again if i close the VPN the website again just loading and loading. Please help me to fixed this issue @amyegan

**Reply by amyegan** [reply]
I need to know more about the network where it is failing (ISP, country, firewall, etc). That will help us figure out what is going wrong.

**Reply by emon3455** [reply]
My public IP address : 103.87.249.198
Country: Bangladesh
firewall: 
![image](https://github.com/vercel/community/assets/107018984/d554a721-6db1-4f7a-82a4-59b875c263ad)
i am using windows-10.
Location: (Dhaka, Bangladesh)
@amyegan

**Reply by emon3455** [reply]
@amyegan is there any update!?
